pkg/webhook/servicecatalog/clusterservicebroker/mutation/handler.go

```diff
@@ -22,9 +22,9 @@ import (
 	"net/http"
 
 	sc "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	webhookutil "github.com/kubernetes-incubator/service-catalog/pkg/webhook/util"
```

Comment:
have you introduced util package?

Reply:
as disscussed will be moved to `pkg/webhookutil`